src/streamcompactionops.cu

```diff
@@ -482,8 +482,8 @@ gdf_error gpu_concat(gdf_column *lhs, gdf_column *rhs, gdf_column *output)
 			size_t prev_len = get_last_byte_length(lhs->size);
 			size_t curr_len = get_right_byte_length(rhs->size, right_num_chars - 1,  prev_len);
 			last_byte[0] = last_byte[0] << curr_len;
-            last_byte[0] = last_byte[0] >> curr_len;
-			size_t last_right_byte_length = rhs->size - GDF_VALID_BITSIZE * (right_num_chars - 1);
+			last_byte[0] = last_byte[0] >> curr_len;
+			//size_t last_right_byte_length = rhs->size - GDF_VALID_BITSIZE * (right_num_chars - 1);
```

Comment:
Yes I hesitated over this. However, since I didn't write this code, I was hesitant to delete it. However, I suppose it can always be recovered from the file history, so I will remote it. Thanks for the nudge.